apps/web/index.html

```diff
@@ -12,9 +12,7 @@
     
     
     <!-- Icon Library -->
-    <!-- <script src="path/to/dist/feather.js"></script> -->
-    <!-- <script src="https://unpkg.com/feather-icons"></script> -->
-    <script src="https://unpkg.com/feather-icons"></script>
+    <script src="https://cdn.jsdelivr.net/npm/feather-icons/dist/feather.min.js"></script>
```

Comment:
Does this mean we don't need the feather package in our `package.json` file?

Reply:
Yeah, I removed that file as well.